config/config.js

```diff
@@ -4,6 +4,19 @@ var token = require("./slack_token.js").token;
 var private_key = require("./service_account_key.js").key;
 
 var config = {
+    // Unfortunately this all has to be at the top level due to sequelize-cli
```

Comment:
Looking at how this is used, I don't see why this can't be organized in an inner object that gets passed to sequelize.

Can you detail this limitation further?